staging/src/k8s.io/kubectl/pkg/util/util.go

```diff
@@ -47,6 +47,9 @@ func HashObject(obj runtime.Object, codec runtime.Codec) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	// TODO: finish removing existing md5 usage
+	// https://github.com/kubernetes/kubernetes/issues/129652
+	//nolint:forbidigo
 	return fmt.Sprintf("%x", md5.Sum(data)), nil
```

Comment:
This utility doesn't actually seem to be used anywhere by us:
https://cs.k8s.io/?q=%5C.HashObject&i=nope&literal=nope&files=&excludeFiles=&repos=

... however it is exported

Reply:
I'm fine with deleting it. Even though it's an exported function it's small enough that anyone relying on its functionality can easily copy&paste those 4 lines. Digging through history it seems this was used for rolling updates, which initially were coupled inside kubectl, but eventually the entire logic migrated over to deployment controller.

Reply:
done + rebased